FreeRTOS/FreeRTOS-Plus-TCP
# Update release script to add cleanup step

Description
-----------
This PR updates the release actions script by adding the Cleanup step to delete the version number branch created in the repository by the SBOM generator.

Test Steps
-----------
[Tested on fork.](https://github.com/tony-josi-aws/FreeRTOS-Plus-TCP/actions/runs/6690793658)

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
